src/main/java/org/opentripplanner/middleware/triptracker/TravelerLocator.java

```diff
@@ -115,6 +114,37 @@ public static TripInstruction alignTravelerToTrip(TravelerPosition travelerPosit
         return null;
     }
 
+    /**
+     * Check that the current position is not passed the "next step". This is to prevent an instruction being provided
+     * for a step which is behind the traveler, but is within radius.
+     */
+    private static boolean isPositionPassedStep(TravelerPosition travelerPosition, Step nextStep) {
```

Comment:
We can probably add the compass code here to resolve this? Also possible typo `past`?

Reply:
Addressed typo. Thanks.